packages/vite/src/node/ssr/ssrTransform.ts

```diff
@@ -321,10 +321,12 @@ function walk(
                 !isStaticPropertyKey(child, parent) &&
                 // do not record if this is a default value
                 // assignment of a destructuring variable
-                !(
-                  parent &&
-                  parent.type === 'AssignmentPattern' &&
-                  parent.right === child
+                (
+                  !parent || 
```

Comment:
The trailing whitespace tells me that you didn't run the formatter
Please run `pnpm run format` and commit the file newly
#my-inner-linter-is-triggered :rofl: 

---

beside that, I think the condition is now much harder to read :thinking: 
Can it somehow get simplified? If not, it's also okay :shrug:

Reply:
NeoVim was actually showing it to me but I didn't know pnpm run format would solve it without any work on my side. Thank you!
Else I tried to make the if statement easier to read by returning early from the enter function.